server/controllers/finance/debtors/index.js

```diff
@@ -36,8 +36,8 @@ function list(req, res, next) {
   const sql = `
     SELECT BUID(d.uuid) AS uuid, BUID(d.group_uuid) AS group_uuid,
       d.text, map.text as hr_entity
-    FROM debtor d
-    JOIN entity_map map ON map.uuid = d.uuid;
+    FROM debtor d 
```

Comment:
Some debtors have not an entity_map so ```LEFT JOIN``` gave me every row I had also this problem with creditor and I did the same. I saw the triggers every thing seems well.